### PR TITLE
Modify Teams Page UI

### PR DESCRIPTION
**Pull request by aaditkamat**
## Status
**READY**

## Migrations
NO

## Description
Update the UI for the teams page by making table columns wider and the team panel width same as the table width

### CODE REVIEW

**Comment by bokwoon95** [comment]
Hi, the elements still seem to be misaligned with the table on my end, except everything's slightly bigger

Before
Screenshot 2019-06-04 at 9 39 24 AM

After
Screenshot 2019-06-04 at 9 41 40 AM

**Comment by aaditkamat** [comment]
Not sure why this is happening because when I apply the changes to the teams page on Skylab by modifying the HTML using dev tools, there seems to be no issue.

Screenshot 2019-06-05 at 6 59 26 PM

File: app/views/teams/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% content_for :main_content do %>
     <% javascript 'teams.js' %>
-    <div class="panel panel-info" style="width: min-content">
```
Comment (aaditkamat) [inline app/views/teams/index.html.erb]: Alternatively I could change the `width` attribute to `max-content` and let the size of the buttons remain as they were

File: app/views/teams/_teams_table.html.erb [hunk 2]
```diff
@@ -18,30 +18,18 @@
...
-          <% else %>
-            <%= team.get_team_submission_status(team.get_own_submissions_in_order[3]) %>
-          <% end %>
-        </td>
+          <% for i in 1..3 do %>
+            <td>
+            <% submission = team.get_own_submissions_in_order[i] %>
+            <%if team.get_team_submission_status(submission) == "Submitted" %>
+              <a class="btn btn-primary btn-xs" href="<%= milestone_team_submission_path(submission.milestone_id, submission.team_id, submission.id)%>"> View Submission </a>
+            <% elsif team.get_team_submission_status(submission) == "Late" %>
```
Comment (aaditkamat) [inline app/views/teams/_teams_table.html.erb]: Adding a late submission button for #666

**Comment by aaditkamat** [comment]
Now the teams page looks like this: 
Screenshot 2019-06-05 at 9 42 16 PM

Where the blue buttons are for those teams that submitted on time and yellow buttons for those who submitted it late. 

Screenshot 2019-06-05 at 9 42 38 PM

I have changed the link for the blue/yellow buttons to point to the page for creating a new peer evaluation instead of the page to view the submission because there is already a button for the latter embedded within the former.

**Comment by bokwoon95** [comment]
Okay I think the misalignment is something that can only be replicated on the server, so it's not easy to tackle. I'm going to merge this in for now.
Screenshot 2019-06-08 at 11 59 22 AM

**Pull request merged**